scan/src/pages/ValidatorIndexPage.re

```diff
@@ -75,100 +75,125 @@ let kvRow = (k, v: value_row_t) => {
 };
 
 [@react.component]
-let make = (~address, ~hashtag: Route.validator_tab_t) => {
-  let isActive = true;
+let make = (~address, ~hashtag: Route.validator_tab_t) =>
+  {
+    let%Opt info = React.useContext(GlobalContext.context);
+    let%Opt validator =
+      info.validators
+      ->Belt_List.keep(({operatorAddress}) => operatorAddress->Address.isEqual(address))
```

Comment:
Use `ValidatorHook.get(address)`

Reply:
We use info.validators because we need to compute totalPower from all validators